kafka-plugins-0.10/pom.xml

```diff
@@ -11,22 +11,45 @@
 
   <name>Apache Kafka 0.10 plugins</name>
   <artifactId>kafka-plugins</artifactId>
-  <version>${project.parent.version}-0.10.2.0</version>
+  <version>${project.parent.version}-${kafka10.version}</version>
```

Comment:
I'd not use "10" in the name as we drop "8". 10 is universal client, but we may want to go to later version eventually

Reply:
I've consolidated everything into a `kafka-plugins-client` module